### ISSUE TITLE
Process to create, label and show a new chat

### ISSUE THREAD

**Issue by MarkGOC** [dto-btn/ssc-assistant#558]
After a user has clicked the new chat button we show them our welcome screen for creating new chats but only after they put in a prompt hit submit, our naming code runs and names the chat.  Is the new chat created and added to the menu on the left.

We can't have chats with no label for UX and WCAG reasons.
